scripts/run-webservice-script.sh

```diff
@@ -15,6 +15,6 @@ chmod u+x dockstore-webservice.jar
 psql -h localhost -c "create user dockstore with password 'dockstore' createdb;" -U postgres
 psql -h localhost -c "ALTER USER dockstore WITH superuser;" -U postgres
 psql -h localhost -c 'create database webservice_test with owner = dockstore;' -U postgres
-psql -h localhost -f travisci/db_dump.sql webservice_test -U postgres
+psql -h localhost -f travisci/${DB_DUMP:-db_dump.sql} webservice_test -U postgres
 java -jar dockstore-webservice.jar db migrate -i 1.5.0,1.6.0,1.7.0,add_service_1.7.0,1.8.0,1.9.0,1.10.0,alter_test_user_1.10.2,1.11.0,1.12.0 travisci/web.yml
```

Comment:
Do you need most of these? I know they don't hurt, but if if most of the migrations have already been applied...

Reply:
`db_dump.sql` needs them, `smoke_test_db.sql` doesn't, so it's easier to leave as-is for the moment.